dynamodump/dynamodump.py

```diff
@@ -49,10 +49,25 @@
 SCHEMA_FILE = "schema.json"
 THREAD_START_DELAY = 1  # seconds
 
-json.JSONEncoder.default = lambda self, obj: (
-    obj.isoformat() if isinstance(obj, datetime.datetime) else None
-)
+import base64
 
+def encoder(self, obj):
+    if isinstance(obj, datetime.datetime):
+        return obj.isoformat()
+
+    if isinstance(obj, bytes):
+        return base64.b64encode(obj).decode("utf-8")
+
+    return json.JSONEncoder.encoder(self, ob)
```

Comment:
```suggestion
    return json.JSONEncoder.encoder(self, obj)
```
I think there is a typo here.

Reply:
Thanks for pointing this out, not sure why this passed my manual testing in the past 🤔